config/sara/sara.yml

```diff
@@ -1,5 +1,5 @@
 # Version
-version: v1.27.0
+version: v1.28.0
```

Comment:
Rebase issue?

Reply:
This should be fixed in https://github.com/SaraAlert/SaraAlert/pull/823/commits/e541f149fac362f6255473ec723c7cbb53016987.